### PR TITLE
Updated JUnit tests

### PR DESCRIPTION
**Pull request by lohjooh**

### CODE REVIEW

File: src/main/java/seedu/clialgo/file/Code.java [hunk 1]
```diff
@@ -73,4 +73,19 @@ public void setImportance(int importance) {
...
+            return true;
+        }
+        if (object == null || this.getClass() != object.getClass()) {
+            return false;
+        }
+        Code checkedCode = (Code) object;
+        boolean sameName = this.getName().equals(checkedCode.getName());
+        boolean samePath = this.getPath().equals(checkedCode.getPath());
+        boolean sameTopic = this.getTopic().equals(checkedCode.getTopic());
+        boolean sameImportance = this.getImportance() == checkedCode.getImportance();
```
Comment (heejet) [inline src/main/java/seedu/clialgo/file/Code.java:88]: I think the names should be `isSameName` `isSamePath` etc

File: src/main/java/seedu/clialgo/file/Note.java [hunk 2]
```diff
@@ -81,4 +81,19 @@ public int getImportance() {
...
+            return true;
+        }
+        if (object == null || this.getClass() != object.getClass()) {
+            return false;
+        }
+        Note checkedCode = (Note) object;
+        boolean sameName = this.getName().equals(checkedCode.getName());
+        boolean samePath = this.getPath().equals(checkedCode.getPath());
+        boolean sameTopic = this.getTopic().equals(checkedCode.getTopic());
+        boolean sameImportance = this.getImportance() == checkedCode.getImportance();
```
Comment (heejet) [inline src/main/java/seedu/clialgo/file/Note.java:96]: same as above

File: src/test/java/seedu/clialgo/storage/FileManagerTest.java [hunk 3]
```diff
@@ -13,6 +14,8 @@
 
 public class FileManagerTest {
 
+    private final String path = ".\\testdata";
```
Comment (heejet) [inline src/test/java/seedu/clialgo/storage/FileManagerTest.java:17]: If this is a constant it should be in caps

**Review by heejet — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/clialgo/storage/FileManagerTest.java [hunk 4]
```diff
@@ -23,6 +26,8 @@ public void deleteAll(File pathToFolder) {
             for (File f : files) {
                 if (!f.delete()) {
                     System.out.println("Delete failed");
+                } else {
+                    System.out.println("Delete success");
```
Comment (nikkiDEEE) [inline src/test/java/seedu/clialgo/storage/FileManagerTest.java:30]: perhaps printing could be done via the Ui class?

Comment (lohjooh) [reply src/test/java/seedu/clialgo/storage/FileManagerTest.java:30]: The test printing are not meant for interacting with the user, rather it is used to debug the JUnit test themselves

File: src/main/java/seedu/clialgo/file/Code.java (hunk 1 above)
Comment (lohjooh) [reply src/main/java/seedu/clialgo/file/Code.java]: You're right, the changes have been made

File: src/main/java/seedu/clialgo/file/Note.java (hunk 2 above)
Comment (lohjooh) [reply src/main/java/seedu/clialgo/file/Note.java]: same as above

File: src/test/java/seedu/clialgo/storage/FileManagerTest.java (hunk 3 above)
Comment (lohjooh) [reply src/test/java/seedu/clialgo/storage/FileManagerTest.java]: Done, the changes have been made to BufferTest, SingleFileTest, FileEncoderTest and FileManagerTest

File: src/test/java/seedu/clialgo/storage/FileManagerTest.java (hunk 4 above)
Comment (lohjooh) [reply src/test/java/seedu/clialgo/storage/FileManagerTest.java:30]: I could make the changes if necessary, however, what do you think?

**Review by heejet — APPROVED** [review]

**Pull request merged**